cedar-policy-core/src/evaluator/err.rs

```diff
@@ -63,8 +63,8 @@ pub enum EvaluationError {
         actual: Type,
     },
 
-    /// Wrong number of arguments to an extension function
-    #[error("wrong number of arguments to {function_name}: expected {expected}, got {actual}")]
+    /// Wrong number of arguments provided to an extension function
+    #[error("wrong number of arguments provided to extension function {function_name}: expected {expected}, got {actual}")]
```

Comment:
"\`{function_name}\`"?

Reply:
My convention has been to use backticks for customer-controlled strings. `function_name` is something we control (e.g., `ip`, `lessThan`), and `expected` and `actual` are just numbers, so I don't think backticks are necessary.